build/circle-deps.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -ex
+
+if [ "$1" = "docker" ]; then
+    cmds=$(grep '^cmd' GLOCKFILE | grep -v glock | awk '{print $2}')
+
+    # Restore previously cached build artifacts.
+    time go install github.com/cockroachdb/build-cache
+    time build-cache restore . .:race,test ${cmds}
+    # Build everything needed by circle-test.sh.
+    time go test -race -v -i ./...
+    time go install -v ${cmds}
+    time make install
+    time (cd acceptance; go test -v -c -tags acceptance)
+    # Cache the current build artifacts for future builds.
+    time build-cache save . .:race,test ${cmds}
+
+    exit 0
+fi
+
+gopath0="${GOPATH%%:*}"
+cachedir="${gopath0}/pkg/cache"
+tag="20150719-133944"
+
+mkdir -p "${cachedir}"
+du -sh "${cachedir}"
+ls -lh "${cachedir}"
+
+if ! docker images | grep -q "${tag}"; then
+    # If there's a base image cached, load it. A click on CircleCI's "Clear
+    # Cache" will make sure we start with a clean slate.
+    rm -f "$(ls ${cachedir}/builder*.tar 2>/dev/null | grep -v ${tag})"
```

Comment:
also `-a` is an illegal optional in BSD find. I guess it would have to be:
```
find "${cachedir}" -type f -name 'builder*.tar' -not -name "*${tag}*" -delete
```
if that works in GNU find.